### ISSUE TITLE
Business requirements - New app styling

### ISSUE THREAD

**Issue by annapakula** [binawork/OnlineOnboarding#228]
- [ ] change background
- [ ] add logo
- [ ] change pages' layouts
- [ ] change cards' styles
- [ ] change byttons' styles

**Comment by maggienow** [comment]
@annapakula - Ania w wersji mobilnej , nowego designu, zniknął nam navi bar.Czy mozesz zajrzeć proszę i sprawdzić w czym rzecz? 

![image](https://user-images.githubusercontent.com/67949340/118270305-27d62c00-b4c0-11eb-9221-ba71f672d528.png)

**Comment by maggienow** [comment]
@annapakula - Adam Ci wyjasni, co tam potrzeba.

**Comment by annapakula** [comment]
@maggienow tak, zniknął, bo w nowym widoku go nie ma. Na potrzeby hybrydowego stylu (starego i nowego) trzeba by przywrócić hamburger menu.

**Comment by annapakula** [comment]
@maggienow @adam-sas-on  na razie zrobiłam tak:
![image](https://user-images.githubusercontent.com/51723246/118327532-039d3e00-b506-11eb-9a45-c3f17e54e9e7.png)
![image](https://user-images.githubusercontent.com/51723246/118327682-0dbf3c80-b506-11eb-8235-315cb62e164b.png)

**Comment by maggienow** [comment]
@annapakula Ania,super!!! tak chyba musi zostac ten hamburger menu ;)

**Comment by annapakula** [comment]
Tak jak wspominałam - oddałam laptopa do serwisu. Ostatnio nie udało mi się go uruchomić. Do 2 czerwca powinnam wiedzieć, co z nim. Bez niego nie ruszę dalej z tematem.

**Comment by maggienow** [comment]
@Michal-Roszyk please send new designed logo to Adam

**Comment by maggienow** [comment]
> Tak jak wspominałam - oddałam laptopa do serwisu. Ostatnio nie udało mi się go uruchomić. Do 2 czerwca powinnam wiedzieć, co z nim. Bez niego nie ruszę dalej z tematem.

Oki.to czekamy, az bedziesz miala dostep do komputera

**Comment by maggienow** [comment]
Please take into consideration #242

**Comment by maggienow** [comment]
@annapakula - Ania testowalismy z Adamem to wylogowywanie i ani u mnie ,ani u Adama nie działa wylogowywanie. Ja testowalam na Chrome i na Edge.
